### PR TITLE
feat: Added FAQ entry for updating npm Enterprise license

### PR DESCRIPTION
**Pull request by mahikhond**

### CODE REVIEW

File: troubleshooting/faq.md [hunk 1]
```diff
@@ -161,6 +162,18 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## How do I upgrade npm Enterprise license?
```
Comment (bcoe) [inline troubleshooting/faq.md]: Let's change this to:

"How do I update my npm Enterprise license?"

And switch `upgrade` to update elsewhere.

File: troubleshooting/faq.md [hunk 2]
```diff
@@ -161,6 +162,18 @@ So it’s important to understand the different storage driver options available
...
+
+You can upgrade npme license by following below steps:
```
Comment (bcoe) [inline troubleshooting/faq.md]: "by following the steps below"

File: troubleshooting/faq.md [hunk 3]
```diff
@@ -161,6 +162,18 @@ So it’s important to understand the different storage driver options available
...
+
+1. Run `npme update-license` on your npm Enterprise server.
+
+2. It will prompt you for billing email, enter your registered billing email.
+
+3. Then it will prompt you for license key, enter your license key.
+
+You can get license key by purchasing npm Enterprise license from [here](https://www.npmjs.com/enterprise/license)
```
Comment (bcoe) [inline troubleshooting/faq.md]: I like that you direct people to the license registration page 👍

**Review by bcoe — CHANGES_REQUESTED** [review]
sounding good 👍 just a couple updates for you.

File: troubleshooting/faq.md [hunk 4]
```diff
@@ -161,6 +162,18 @@ So it’s important to understand the different storage driver options available
...
+
+You can update npm Enterprise license by following the steps below:
+
+1. Run `npme update-license` on your npm Enterprise server.
+
+2. It will prompt you for billing email, enter your registered billing email.
+
+3. Then it will prompt you for license key, enter your license key.
+
+You can get license key by purchasing npm Enterprise license from [here](https://www.npmjs.com/enterprise/license)
```
Comment (arobson) [inline troubleshooting/faq.md]: Minor change in wording/grammar: "You can get a license key by purchasing an npm Enterprise license on our [license page](https://www.npmjs.com/enterprise/license)."

**Review by arobson — CHANGES_REQUESTED** [review]

**Review by arobson — APPROVED** [review]

**Pull request merged**